src/rules/prefer-in-document.js

```diff
@@ -48,6 +51,24 @@ function usesToHaveLengthZero(matcherNode, matcherArguments) {
   );
 }
 
+/**
+ * Extract the DTL query identifier from a call expression
+ *
+ * <query>() -> <query>
+ * screen.<query>() -> <query>
+ */
+function getDTLQueryIdentifierNode(callExpressionNode) {
+  if (!callExpressionNode || callExpressionNode.type !== "CallExpression") {
+    return;
```

Comment:
I'd prefer we explicitly return `null` here:

```suggestion
    return null;
```